File: pinyougou-shop-web/src/main/webapp/js/service/goodService.js
app.service("goodService", function ($http){

    //商品添加
    this.add = function (goods) {
        return $http.post("../good/save.do",goods);
    }
    //商品列表，只展示当前商家自己的
    this.findPage = function (page,size) {
        return $http.get("../good/findPage.do?page="+page+"&size="+size);
    }
    //查询
    this.search = function (tbGoods,page,size) {
        return $http.post("../good/search.do?page="+page+"&size="+size,tbGoods);
    }
});

File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/GoodService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.group.pojo.Goods;
import com.pinyougou.pojo.TbGoods;
import entity.PageResult;

import java.util.List;

public interface GoodService {
    /**
     * 商品添加
     * @param goods
     */
    public void add(Goods goods);
    //商品列表展示
    public PageResult findPage(String selleId,int pageNum,int pageSize);
    //查询商品列表
    public PageResult search(TbGoods goods,int pageNum,int pageSize);

}
